### DISCUSSION TITLE
Issue with Another Auth Model

### DISCUSSION THREAD

**Discussion by jerry4rahul** [spatie/laravel-permission#2086] [Q&A]
**Describe the bug**

When we use `User` Model then it working fine.
If we want to get the Auth User Roles by using
`Auth::user()->roles`
It successfully returning the assigned roles of user.

But the problem is Let say if we user Another Authenticable Model called `Employee`
After assigning the role we are getting empty string

As I checked by finding user manually and using `with` it returns the user with roles
`User::with('roles')->find(1);`

Waiting to get this problem solved.

**Versions**
- spatie/laravel-permission package version: v5
- illuminate/framework package: v8

PHP version: 7.4

**To Reproduce**
-- Use Another Authenticatable Model
-- Login with user
-- Use `Auth::user()->roles`
-- Return will be an empty string

**Comment by erikn69** [comment]
Very probably you configured something wrong, create a minimal application in a clean installation where you demonstrate the problem, surely when you do that you will find what you configured wrong
